spec/models/student_spec.rb

```diff
@@ -2,6 +2,17 @@
 
 RSpec.describe Student, :type => :model do
 
+  it "returns a student's full name" do
+    student = build(:student, title: "Mr", first_name: "John", middle_name: "Adam", last_name: "Smith")
+    expect(student.full_name).to eq("Mr John Adam Smith")
+  end
```

Comment:
consider moving `student` to a [let](https://rubydoc.info/gems/rspec-core/RSpec/Core/MemoizedHelpers/ClassMethods#let-instance_method) binding

```suggestion
  let(:student) { build(:student, ...) }
  it "returns a student's full name" do
    expect(student.full_name).to eq("Mr John Adam Smith")
  end
```

Reply:
Yeah I like that a lot better